src/main/java/org/folio/rest/impl/OrderTemplatesAPI.java

```diff
@@ -28,7 +31,17 @@ public void getOrdersStorageOrderTemplates(String query, String totalRecords, in
   @Override
   @Validate
   public void postOrdersStorageOrderTemplates(OrderTemplate entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    PgUtil.post(ORDER_TEMPLATES_TABLE, entity, okapiHeaders, vertxContext, PostOrdersStorageOrderTemplatesResponse.class, asyncResultHandler);
+    validateCustomFields(vertxContext, okapiHeaders, entity)
+        .compose(
+            v ->
```

Comment:
please use the same code style as in other places, many people contributing to this module, and we should have some standard, to keep it we need to follow already written code

Reply:
I understand the importance of thorough logging, but I'm not entirely convinced that adding an additional log message would provide a significant benefit.

The exception is named `CustomFieldValidationException`, indicating an error related to custom fields and validation. It is logged by the calling API methods, for example [here](https://github.com/folio-org/mod-orders-storage/pull/380/files#diff-d16050af5f2ff623e9f6459f8f6e3d126a10421ba6202067162d5e2032e061d7R75-R77). The response generated by `buildErrorResponse` provides detailed information about the validation error(s).

Considering the provided context and the existing logging mechanisms, do you still think it's necessary to include the additional log message?